target/linux/generic/hack-5.10/921-add-constants-in-include-linux-mtd-partitions-h.patch

```diff
@@ -0,0 +1,17 @@
+--- /dev/null
++++ b/include/dt-bindings/mtd/partitions.h
+@@ -0,0 +1,14 @@
++/* SPDX-License-Identifier: GPL-2.0-only */
++/*
++ * Device Tree constants identical to those in include/linux/mtd/partitions.h
++ */
++
++#ifndef _DT_BINDINGS_MTD_PARTITIONS_H
++#define _DT_BINDINGS_MTD_PARTITIONS_H
++
++#define MTDPART_OFS_RETAIN	(-1) (-3)
```

Comment:
Hi! 
It isn't a good idea to put two separated values in one definition. It's confusing when reading device tree.
Could you put the original 4 values here instead?
```
#define MTDPART_OFS_RETAIN      (-3)
#define MTDPART_OFS_NXTBLK      (-2)
#define MTDPART_OFS_APPEND      (-1)
#define MTDPART_SIZ_FULL        (0)
```

Reply:
The latest approach may satisfy you.